### DISCUSSION TITLE
per page css

### DISCUSSION THREAD

**Discussion by vb8448** [unpoly/unpoly#552] [Q&A] [answered]
Hi, 

i have some per page css file and currently, I'm putting the `<link rel="stylesheet" href="..." />` in the body of the response instead of the head.

It works, but I was wondering if it is the best approach.

Thanks.

**Comment by vb8448** [comment]
I just saw release notes of the version 3.5, for what i understand the recommended way is to handle the `up:assets:changed` event, right?

**Reply by triskweline** [reply, marked as answer]
The `up:assets:changed` event is dispatched when styles or scripts in the `<head>` change. It does not look for assets in the `<body>`.

It's perfectly OK to include a stylesheet in a fragment. It will be active as long as the fragment is visible on the page. If the fragment gets removed, the stylesheet will be unloaded.
